src/NavItem.js

```diff
@@ -23,32 +26,37 @@ const NavItem = React.createClass({
 
   render() {
     let {
+        role,
+        linkId,
         disabled,
         active,
         href,
         title,
         target,
         children,
+        'aria-controls': ariaControls,  // eslint-disable-line react/prop-types
```

Comment:
@Cellule has fixed it https://github.com/yannickcr/eslint-plugin-react/pull/124 :sparkle: